components/list/README.md

```diff
@@ -168,6 +169,7 @@ Where the parameters correspond to the slots of `d2l-list-item`:
 - `disabled` (Boolean): Whether or not the checkbox is disabled
 - `draggable` (Boolean): Whether or not the item is draggable
 - `key` (String): Value to identify item if selectable
+- `label` (String): The hidden label for the checkbox if selectable
 - `selectable` (Boolean): Indicates a checkbox should be rendered for selecting the item
 - `selected` (Boolean): Whether the item is selected
 - `slim` (Boolean): Whether to render the item with reduced whitespace
```

Comment:
I will add something for this.  I kinda think this doc needs to be completely rewritten to make it easier for consumers to follow, and maybe that happens when we start setting up the readmes for the new doc stuff that @margaree and @GeorgePlukov are working on.  This might be a good example a complicated component to document.

Reply:
Agreed yeah. I'm good if you want to wait for that refactor or do it separately, but it's definitely hard to follow as it is now.

Reply:
Sounds good.  I added a section for the `d2l-list-header` since I'd forgotten that.